MyPiggyBank.Data/Model/Resource.cs

```diff
@@ -6,9 +6,12 @@ namespace MyPiggyBank.Data.Model {
     public class Resource
     {
         public Guid Id { get; set; }
-        public string UserEmail { get; set; }
+        public Guid UserId { get; set; }
         public string Name { get; set; }
         public decimal Value { get; set; }
         public string Currency { get; set; }
```

Comment:
Jeśli "Value" nie może być nullem to currency też. Zmiana na required. Nawet jeśli będzie wartość 0 to i tak musimy mieć jakąś walutę.